utils/get_test_status.py

```diff
@@ -0,0 +1,25 @@
+from typing import Dict
+
+import urllib3
+from bs4 import BeautifulSoup
+from urllib3.exceptions import InsecureRequestWarning
+
+urllib3.disable_warnings(category=InsecureRequestWarning)
+
+
+def _get_test_status(self, id_: int, index: str) -> Dict[str, bool]:
```

Comment:
shouldn't this return a List of Tuples instead?

Reply:
for some cases index may not be sequence